hercules-ci-agent/hercules-ci-agent/Hercules/Agent/Scribe.hs

```diff
@@ -24,7 +25,7 @@ withHerculesScribe :: App a -> App a
 withHerculesScribe m = withHerculesScribe' V3 $ \scribe -> do
   logEnv0 <- getLogEnv
   logEnv1 <- liftIO
-    $ registerScribe "hercules" scribe defaultScribeSettings logEnv0
+    $ registerScribe "hercules" scribe (defaultScribeSettings & scribeBufferSize .~ 1000000 ) logEnv0
```

Comment:
I've deleted this scribe in https://github.com/hercules-ci/hercules-ci-agent/pull/111

I don't think it's a good idea to make our backend sweat on this feature.